docs/data/stashcache/install-origin.md

```diff
@@ -11,108 +11,134 @@ to export its data to the StashCache data federation.
-monitoring data about the origin. To simplify installation, OSG provides convenience RPMs that install all required
+The origin service consists of one or more XRootD daemons and their dependencies for the authentication infrastructure.
+To simplify installation, OSG provides convenience RPMs that install all required
 software with a single command:
 
 ```console
-root@host # yum install stashcache-origin-server
+root@host # yum install --enablerepo=osg-testing stash-origin
 ```
 
-For this installation guide, we assume that the data to be exported to the federation is mounted at `/stash`
+!!! note
+    This document covers a completely overhauled stash origin service based on XCache 1.0.2;
+    while this has not yet been released into the OSG production repository,
+    we strongly recommend using this version for configuration and accounting improvements.
+
+For this installation guide, we assume that the data to be exported to the federation is mounted at `/mnt/stash`
 and owned by the `xrootd:xrootd` user.
 
 Configuring the Origin Server
 -----------------------------
 
-The `stashcache-daemon` package provides a default configuration file,
-`/etc/xrootd/xrootd-stashcache-origin-server.cfg`, which must be customized for your origin.
+The `stash-origin` package provides a default configuration files in
+`/etc/xrootd/xrootd-stash-origin.cfg` and `/etc/xrootd/config.d`.  Administrators may provide additional configuration
+by placing files in `/etc/xrootd/config.d` of the form `/etc/xrootd/config.d/1*.cfg` (for directives that need to
+be processed BEFORE the OSG configuration) or `/etc/xrootd/config.d/9*.cfg` (for directives that are processed
+AFTER the OSG configuration).
+
+You _must_ configure every variable in `/etc/xrootd/10-common-site-local.cfg` and `/etc/xrootd/10-origin-site-local.cfg`.
+
+The mandatory variables to configure in `10-common-site-local.cfg` are:
 
-The most common lines to customize are:
+* `set rootdir = /mnt/stash`: the mounted filesystem path to export.
+  This document refers to this path as `/mnt/stash`.
+* `set resourcename = YOUR_RESOURCE_NAME`: the resource name registered with OSG.
 
-* `oss.localroot /stash`: Change the `localroot` to the location where your data is mounted on
-  the origin server; default is `/stash`.
-* `all.export /<YOUR VO>`: A sub-directory within the `localroot` directory that will be exported.
-  If multiple directories must be exported, you may specify `all.export` multiple times.
+The mandatory variable to configure in `10-origin-site-local.cfg` is:
 
-For example, if the HCC VO would like to set up an origin server exporting from the mountpoint `/mnt/bigdata`,
-but only export the subdirectories `/mnt/bigdata/hcc/bio/datasets` and `/mnt/bigdata/hcc/hep/generators`,
-they would use the following configuration:
+* `set originexport = /VO`: the directory relative to `rootdir` that is the top of the exported namespace
```

Comment:
We shouldn't have lists of only one item. Maybe consider making all of the required variables into a table with name, file, and description as the columns.